hal/lib/athena/FRCDriverStation.cpp

```diff
@@ -16,7 +16,7 @@
 #include "HAL/cpp/priority_mutex.h"
 
 struct HAL_JoystickAxesInt {
-  uint16_t count;
+  int16_t count;
```

Comment:
This technically only ever holds a count up 64, so this could go either way. We should only change this if we change the definitions in DriverStation.h  as well. The only one in DriverStation.h we need to be carfeful changing is HAL_JoystickButttons:buttons, as that is a bitfield value and the last bit is definitely used. But its always bit shifted out, so it being an int would be fine, and java uses a signed int anyway. I'd ask @PeterJohnson what his opinion is.